datasets/songs_custom_delimiter.csv

```diff
@@ -0,0 +1,54 @@
+id;title;album;artist;genre;country;released;duration;released-timestamp;duration-float
```

Comment:
I think we could reduce this csv file to a few lines :)

Reply:
Yes! But thank @sanders41, the tests have been missing from the package build since #367. In case you wonder